gcsfs/dask_link.py

```diff
@@ -1,5 +1,10 @@
+import logging
+
 from .core import GCSFileSystem
 
+logging.basicConfig()
```

Comment:
this should be up to the user - for gcsfs, it gets done if the environment variable is set on import

Reply:
deleted